inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/querybuilder/SPARQLQueryBuilder.java

```diff
@@ -663,19 +663,19 @@ public SPARQLQueryBuilder withLabelMatchingExactlyAnyOf(String... aValues)
         IRI ftsMode = forceDisableFTS ? FTS_NONE : kb.getFullTextSearchIri();
         
         if (FTS_LUCENE.equals(ftsMode)) {
-            addPattern(PRIMARY, withLabelMatchingExactlyAnyOf_RDF4J_FTS(aValues));
+            addPattern(PRIMARY, withLabelMatchingExactlyAnyOfRDF4JFTS(aValues));
```

Comment:
Actually, I find the old name more readable as it chunks it into pieces that are constant (withLabelMatchingExactlyAnyOf) and variable ones (RDF4J and FTS).